docs/docs/advanced-use/release-pipelines.md

```diff
@@ -0,0 +1,155 @@
+# Release Pipelines
+
+:::warning
+
+Release Pipelines is currenlty in alpha phase and is not available to the public.
+
+:::
+
+
+Release Pipelines allow you to define a series of stages that your feature flags progress through automatically. Each stage can have specific triggers and actions that control when and how flags are released.
+
+## Key Benefits
+
+- **Automated Deployments**: Reduce manual intervention by automating flag releases across environments
+- **Standardised Process**: Ensure consistent release protocols across your team
+- **Reduced Risk**: Control the pace of rollouts with customisable conditions
+- **Better Visibility**: Track the progress of your feature releases through each stage
```

Comment:
Enumerated or bullet point list items with a sentence should, from a [grammar perspective](https://www.microsoft.com/en-us/microsoft-365-life-hacks/writing/punctuating-bullet-point-lists), end with a period. In this and other lists in this document, periods would favor the structure readability of the text, IMO.